src/cargo/util/toml/mod.rs

```diff
@@ -803,7 +795,7 @@ impl InheritableFields {
     }
 
     /// Gets the field `workspace.lint`.
-    fn lints(&self) -> CargoResult<manifest::TomlLints> {
+    pub fn lints(&self) -> CargoResult<manifest::TomlLints> {
```

Comment:
imo this shouldn't be in `InheritableFields` anymore because we are no longer treating it like normal inheritance.

Thoughts?

Reply:
It is still "Inheritable", it is just not inherited in the normal way. I could go either way on where it should be